<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgendaMain</class>
 <widget class="QMainWindow" name="AgendaMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../sched.qrc">
    <normaloff>:/agenda.png</normaloff>:/agenda.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>20</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="calButton">
        <property name="text">
         <string>Calendar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="text">
         <string>Hide</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="agenda::EventTable" name="activityList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>30</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>TItle</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>When</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>What</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgenda">
    <property name="title">
     <string>Agenda</string>
    </property>
    <addaction name="actionNewItem"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help...</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMaintenance">
    <property name="title">
     <string>Maintenance</string>
    </property>
    <addaction name="actionClearOld"/>
   </widget>
   <addaction name="menuAgenda"/>
   <addaction name="menuMaintenance"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewItem">
   <property name="text">
    <string>New Item</string>
   </property>
  </action>
  <action name="actionClearOld">
   <property name="text">
    <string>Clear Old</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>agenda::EventTable</class>
   <extends>QTableWidget</extends>
   <header location="global">../../src/event-table.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sched.qrc"/>
 </resources>
 <connections/>
</ui>
